client/src/components/v3/DASRecording.tsx

```diff
@@ -0,0 +1,30 @@
+import { useVideoFeedStatus } from 'api/common/camera';
+import React from 'react';
+import { Badge } from 'react-bootstrap';
+import styles from './DASRecording.module.css';
+
+/**
+ * Shows the recording status for the cameras
```

Comment:
Changed to two buttons rather than the toggle switch as that was a bit complicated to implement

![image](https://user-images.githubusercontent.com/50545432/110918238-05fce380-836f-11eb-9fbd-c31b77062f48.png)

08ef4d4

Reply:
Would prefer a toggle switch, but I now see how getting the current state would be difficult so don't worry about it for now :)